querier.hh

```diff
@@ -51,13 +54,21 @@ auto consume_page(flat_mutation_reader_v2& reader,
 class querier_base {
     friend class querier_utils;
 
+public:
+    struct querier_config {
+        uint32_t tombstone_warn_threshold {0}; // 0 disabled
+        querier_config(uint32_t warn)
+            : tombstone_warn_threshold(warn) {}
+    };
+
 protected:
     schema_ptr _schema;
     reader_permit _permit;
     lw_shared_ptr<const dht::partition_range> _range;
     std::unique_ptr<const query::partition_slice> _slice;
     std::variant<flat_mutation_reader_v2, reader_concurrency_semaphore::inactive_read_handle> _reader;
     dht::partition_ranges_view _query_ranges;
+    std::optional<querier_config> _qr_config;
```

Comment:
I think this shouldn't be optional, just add a default constructor to `querier_config` instead. We do have defaults anyway.

Reply:
thx, fix it.